app/views/declaration/dispatch_location.scala.html

```diff
@@ -62,6 +63,24 @@
       )
     )
 
+      <details class="govuk-details" role="group">
```

Comment:
Do we have test for this section ?

Reply:
What would you test?  It's just a block of static content and I've tested that the lists of countries are correctly produced from the csv files.    There aren't any tests for the details/summary section in fiscal_information either.

One thing worth testing is that the details hide/reveal works in IE/Edge because from memory I don't think this standard html 5 tag is supported in these browsers without some javascript trickery.